subcommands/keys/tuf_crypto.go

```diff
@@ -27,12 +27,26 @@ type tufKeyTypeEd25519 struct{}
 
 const (
 	// These are case insensitive
+	tufRoleNameRoot = "Root"
```

Comment:
I wonder if introducing of the `TUF role` type would be helpful and improve readability.

```
type TufRole string

const (
   Root TufRole = "Root"
   Targets = "Targets"
)
```

Reply:
I would have to cast `TufRole <-> string` in a few places; but I'm lazy and don't want to do that.
The Golang typing models teached me to be lazy the hard way.

Reply:
IMHO `tufRoleNameRoot` is more verbose than just `Root`; I'm tired of one-word const names all over the codebase, which lead to subtle errors over time.